internal/databases/mongo/binary/oplog_reply.go

```diff
@@ -79,7 +79,6 @@ func HandleOplogReplay(ctx context.Context,
 	errs = append(errs, errc)
 
 	for _, errc := range errs {
-		errc := errc
```

Comment:
copyloopvar flags this since go 1.22 changed semantics so that this is no longer necessary